src/main/java/com/crosswordservice/controller/CrosswordController.java

```diff
@@ -0,0 +1,106 @@
+    public List<Question> inputTestData(){
+        Configuration config = new Configuration("test");
+        configurationRepository.save(config);
+
+        List<Question> questions = new ArrayList<>();
+
+        Question quest1 = new Question(config.getId(),"Which language extends Javascript with type safety?","Typescript");
+        Question quest2 = new Question(config.getId(),"How is the system of rules called which defines well-formed expressions?","Syntax");
+        Question quest3 = new Question(config.getId(),"Which loop allows to set the count of iterations in the head?","for-loop");
+        Question quest4 = new Question(config.getId(),"What is the abbreviation of the computing unit in a computer?","CPU");
+        questions.add(quest1);
+        questions.add(quest2);
+        questions.add(quest3);
+        questions.add(quest4);
+        questionRepository.saveAll(questions);
+
+        return questions;
+    }
+
+    @PostMapping("/create-configuration")
+    public Configuration saveConfiguration(@RequestBody Configuration configuration){
+        configurationRepository.save(configuration);
+        return configuration;
+    }
+
+    @PostMapping("/save-all-questions/{name}")
+    public List<Question> saveAllQuestions(@RequestBody List<Question> questions, @PathVariable String name) {
+        Configuration config = configurationRepository.findByName(name);
+        questions.forEach(question -> {
+            question.setInternalId(config.getId());
+        });
+        questionRepository.saveAll(questions);
+        return questions;
+    }
+
+    @PutMapping("/update-all-questions/{name}")
+    public List<Question> updateAllQuestions(@RequestBody List<Question> questions, @PathVariable String name){
+        Configuration config = configurationRepository.findByName(name);
+        if(config == null){
+            throw new ResponseStatusException(HttpStatus.NOT_FOUND,"id not found");
+        }
+        questionRepository.deleteByInternalId(config.getId());
+        questions.forEach(question -> {
+            question.setInternalId(config.getId());
+        });
+        questionRepository.saveAll(questions);
+        return questions;
+    }
+
+    @DeleteMapping("/removeQuestion/{id}")
+    public Question removeQuestion(@PathVariable Long id){
+        Question question = questionRepository.getReferenceById(id);
+        questionRepository.deleteById(id);
+        return question;
+    }
+
+    @GetMapping("/get-all-questions")
+    public List<Question> getAllQuestions() {
+        return (List<Question>) questionRepository.findAll();
+    }
+
+    @GetMapping("/get-questions/{name}")
+    public List<Question> getAllQuestions(@PathVariable String name) {
+        Configuration config = configurationRepository.findByName(name);
+
+        List<Question> questions = questionRepository.findByInternalId(config.getId());
+
+        return questions;
+    }
+
+    @GetMapping("/validateCrossword")
```

Comment:
I think `validate-crossword` is better. Also a GetMapping has no RequestBody. Use PostMapping instead.